tough/src/transport.rs

```diff
@@ -147,7 +147,7 @@ impl Transport for FilesystemTransport {
             ));
         }
 
-        let f = std::fs::File::open(url.path()).map_err(|e| {
+        let f = std::fs::File::open(url.to_file_path().unwrap()).map_err(|e| {
```

Comment:
I forgot we will also need to add an error variant to `error.rs` for this and use snafu's context function like this (instead of unwrapping)

```suggestion
        let f = std::fs::File::open(url.to_file_path().context(error::UrlToFile{ url })?).map_err(|e| {
```